### ISSUE TITLE
Next/Previous Sibling performs very slowly

### ISSUE THREAD

**Issue by jaime10a** [vipzhicheng/logseq-plugin-vim-shortcuts#22]
The functions to go to the next or previous Sibling (shift+j and shift+k) are very useful to move around quickly, but they are very slow. So I tend to just press multiple times j to go down, which is not ideal.
I know it may be a technical limitation because it's not a native function of logseq, but it would be great if it could run faster.
Maybe it is better to ask the logseq team to implement such a function?

**Comment by vipzhicheng** [comment]
Yes, you are right, I can just give the suggestion to Logseq Team for now.

**Comment by jakubkaczor** [comment]
@jaime10a, @vipzhicheng, have you asked the Logseq team? If so, perhaps it is worth referencing here.

**Comment by vipzhicheng** [comment]
Just had a short talk with Logseq team dev about this issue. but no better solution to solve it.
